packages/amplify-java-runtime-provider/src/utils/invoke.ts

```diff
@@ -0,0 +1,16 @@
+import path from 'path';
+import { InvocationRequest} from 'amplify-function-plugin-interface';
+import childProcess from 'child_process';
+export async function invokeResource(request: InvocationRequest , context : any){
+  const shimPath = path.join(request.srcRoot, 'src','Mock','build','libs');
+  const cp = childProcess.spawnSync('java', [ '-jar', 'Mock-all.jar' ,request.handler , request.event], { cwd: shimPath , stdio : 'pipe' , encoding: 'utf-8' });
+
+  let result  = JSON.stringify(cp.stdout);
+  try {
+    result = JSON.parse(result);
+  } catch (err) {
+    context.print.warning('Could not parse function output as JSON. Using raw output.');
+  }
+  return Promise.resolve({result});
```

Comment:
Invoke should return just the return value of the underlying lambda function. returning a strigified version of stdout will include any log lines the lambda printed. Additionally, because it looks like your shim is printing `EVENT: <result>` it will include the EVENT prefix which we don't want. And once result is just the return value of the labmda, you should return `Promise.resolve(result)` (no braces) because it shouldn't be wrapped in another object